dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/authentication/ResetPasswordResourceIntegrationTest.java

```diff
@@ -55,12 +59,13 @@ public void initTest(){
     }
 
     @Test
-    public void testNoSuchUserException() throws DotSecurityException, NoSuchUserException, DotInvalidTokenException, SystemException, PortalException {
+    public void testNoSuchUserException() throws DotSecurityException, NoSuchUserException, DotInvalidTokenException, SystemException, PortalException, DotDataException {
         UserManager userManager = getUserManagerThrowingException( new NoSuchUserException("") );
+        final User user = APILocator.getUserAPI().loadUserById("dotcms.org.1");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "dotcms.org.1" appears 7 times in this file; the first occurrence is on line 64](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)